aspnetcore/grpc/diagnostics.md

```diff
@@ -36,7 +44,7 @@ Check the documentation for your configuration system to determine how to specif
 
 We recommend using the `Debug` level when gathering more detailed diagnostics for your app. The `Trace` level produces very low-level diagnostics and is rarely needed to diagnose issues in your app.
 
-### Sample logging output
+#### Sample logging output
 
 Here is an example of console output at the `Debug` level of a gRPC service:
```

Comment:
in the sample - do you add traceID and parentID to the logs scope? Do you add anything to the logs scope? If so - please write about it.

Reply:
This is scopes added by ASP.NET:
```
info: Microsoft.AspNetCore.Routing.EndpointMiddleware[1]
      => ConnectionId:0HLR1BR0PL1CH 
      => RequestPath:/weatherforecastproxy 
         RequestId:0HLR1BR0PL1CH:00000001, 
         SpanId:|363a800a-4cf070ad93fe3bd8., 
         TraceId:363a800a-4cf070ad93fe3bd8, 
         ParentId:
Executed endpoint 'FrontEndApp.Controllers.WeatherForecastProxyController.Get (FrontEndApp)'
```